scenario_player/utils/configuration/settings.py

```diff
@@ -0,0 +1,66 @@
+from typing import Callable, Optional, Union
+
+import structlog
+
+from scenario_player.constants import TIMEOUT
+from scenario_player.exceptions.config import ScenarioConfigurationError
+from scenario_player.utils import get_gas_price_strategy
+from scenario_player.utils.configuration.base import ConfigMapping
+
+log = structlog.get_logger(__name__)
+
+
+class SettingsConfig(ConfigMapping):
```

Comment:
This is missing support for the `settings.services` section.

Reply:
done.